dev-libs/gobject-introspection/gobject-introspection-1.80.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
```

Comment:
This appears to have some really weird gaps. The commit for this package was authored in January, but the PR was opened in *May*. It is missing crucial requirements for py3.12 support that were committed in February: https://github.com/gentoo/gentoo/pull/34945

These long-running PRs that don't get rebased onto newer ebuilds are starting to become a problem, IMHO.